store/tikv/coprocessor.go

```diff
@@ -326,7 +324,10 @@ func (it *copIterator) work() {
 		task.status = taskRunning
 		it.mu.Unlock()
 		bo := NewBackoffer(copNextMaxBackoff, context.Background())
+		startTime := time.Now()
 		resp, err := it.handleTask(bo, task)
+		coprocessorHistogram.Observe(time.Since(startTime).Seconds())
+		backoffHistogram.Observe(float64(bo.totalSleep) / 1000)
```

Comment:
This is intended, we need task time subtract backoff time to get the pure coprocessor time.